.github/workflows/main.yml

```diff
@@ -28,4 +28,9 @@ jobs:
     - name: Build
       run: npm run build
     - name: Test
-      run: npm run test
\ No newline at end of file
+      run: npm run test
+    - name: Upload coverage
+      uses: codecov/codecov-action@v1
```

Comment:
There is a warning about deprecation of `v1` in their repo: https://github.com/codecov/codecov-action#%EF%B8%8F--deprecation-of-v1

Shouldn't we be using another version?

Reply:
Updated to version 3 on [0348408](https://github.com/HathorNetwork/hathor-wallet-lib/pull/405/commits/03484084206ebde7a101faa877c2cf9247295bd9).